### PR TITLE
[Keyboard] Add Hillside support

### PR DESCRIPTION
**Pull request by mmccoyd**
## Description

Add Hillside support, with default and via keymaps and description. (https://github.com/mmccoyd/hillside)

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/hillside/0_1/config.h [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+#include "config_common.h"
+
+/* key matrix size */
+// Rows are doubled-up
+#define MATRIX_ROWS 8
+#define MATRIX_COLS 6
+
+/* Serial */
+#define SERIAL_USE_MULTI_TRANSACTION
```
Comment (drashna) [inline keyboards/handwired/hillside/0_1/config.h:14]: No longer used. 
```suggestion
```

File: keyboards/handwired/hillside/keymaps/default/README.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
+# Default Keymap
```
Comment (drashna) [inline keyboards/handwired/hillside/keymaps/default/README.md:1]: please rename this file to be lower case.

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/handwired/hillside/0_1/config.h (hunk 1 above)
Comment (mmccoyd) [reply keyboards/handwired/hillside/0_1/config.h]: done.

File: keyboards/handwired/hillside/keymaps/default/README.md (hunk 2 above)
Comment (mmccoyd) [reply keyboards/handwired/hillside/keymaps/default/README.md]: Done. Sorry I missed those.

**Comment by mmccoyd** [comment]
The keymap readme could benefit from some edits, which I will be doing today.

**Comment by mmccoyd** [comment]
Readme tweaks complete.

**Review by drashna — APPROVED** [review]

File: keyboards/handwired/hillside/readme.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+Make example for this keyboard (after setting up your build environment):
+
+    make handwired/hillside/0_1:default
+
+Flashing example for this keyboard:
+
+    make handwired/hillside/0_1:default:flash
+
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
+
```
Comment (fauxpark) [inline keyboards/handwired/hillside/readme.md:24]:
```suggestion
```

File: keyboards/handwired/hillside/readme.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+# Hillside
+
+![hillside](https://imgur.com/XW0rX13.png)
+
+[Hillside](https://github.com/mmccoyd/hillside)
+ is a split ergonomic keyboard with 3x6+4+2 choc-spaced keys with
+ aggressive column stager,
+ a longer thumb arc and a breakoff outer-pinky column.
+
```
Comment (fauxpark) [inline keyboards/handwired/hillside/readme.md:9]:
```suggestion
[Hillside](https://github.com/mmccoyd/hillside) is a split ergonomic keyboard with 3x6+4+2 choc-spaced keys with aggressive column stagger, a longer thumb arc and a breakoff outer-pinky column.
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/handwired/hillside/readme.md (hunk 3 above)
Comment (mmccoyd) [reply keyboards/handwired/hillside/readme.md]: extra line removed.

File: keyboards/handwired/hillside/readme.md (hunk 4 above)
Comment (mmccoyd) [reply keyboards/handwired/hillside/readme.md:9]: This looks like a change and a revert.

**Comment by mmccoyd** [comment]
Ahh. The last was a spelling fix, not a 'join all the lines into one' fix. Thanks.

**Review by noroadsleft — APPROVED** [review]
Looks good to me.

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**